src/test/java/io/codeka/gaia/hcl/HCLParserTest.kt

```diff
@@ -0,0 +1,140 @@
+package io.codeka.gaia.hcl
+
+import io.codeka.gaia.modules.bo.Output
+import io.codeka.gaia.modules.bo.Variable
+import org.apache.commons.io.IOUtils
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import org.springframework.core.io.ClassPathResource
+import java.io.IOException
+import java.nio.charset.Charset
+
+class HCLParserTest {
+
+    private val hclParser = HclParserImpl()
+
+    @Test
+    @Throws(IOException::class)
+    fun parsing_variables_shouldWorkWithVisitor() {
+        // given
+        val fileContent = IOUtils.toString(ClassPathResource("hcl/variables.tf").url, Charset.defaultCharset())
+
+        // when
+        val variables = hclParser.parseVariables(fileContent)
+
+        // then
+        Assertions.assertThat(variables).hasSize(3)
+        val stringVar = Variable("string_var", "string", "a test string var", "foo")
+        val numberVar = Variable("number_var", "number", "a test number var", "42")
+        val boolVar = Variable("bool_var", defaultValue = "false")
+        Assertions.assertThat(variables).contains(stringVar, numberVar, boolVar)
```

Comment:
static imports maybe ?

Reply:
Damn, I always forget that. I tried to find a way to force my IDE to do that for me but I didn't find the right setting.
✅